### ISSUE TITLE
Drag and drop text within the Editor

### ISSUE THREAD

**Issue by Soreine** [GitbookIO/feedback#279]
One should be able to select some text, and drag it to another place to move the text there (same as Cut + Paste).

**Comment by Tracy1105** [comment]
yeah, same problem,  it 's not easy to edit my book, i can not choose the section which i want to edit.
